test/e2e-tests.sh

```diff
@@ -31,39 +31,10 @@ install_dashboard_backend
 #Run the integration tests
 header "Running e2e tests"
 
-kubectl port-forward $(kubectl get pod -l app=tekton-dashboard -o name) 9097:9097 &
+kubectl port-forward $(kubectl get pod --namespace tekton-pipelines -l app=tekton-dashboard -o name)  --namespace tekton-pipelines 9097:9097 &
 dashboardForwardPID=$!
 
-#Apply permissions to be able to curl endpoints 
-kubectl apply -f $tekton_repo_dir/test/test-rbac.yaml
-kubectl apply -f $tekton_repo_dir/test/kaniko-build-task.yaml
-kubectl apply -f $tekton_repo_dir/test/deploy-task-insecure.yaml
-kubectl apply -f $tekton_repo_dir/test/Pipeline.yaml
-
-#API configuration
```

Comment:
Hmm, why remove the manual pipelinerun e2e test? This is still a supported thing - yeah it's deploying an application and a service, and a ksvc, but surely then we should just have another simple role set up for the tests only.